apps/bot/lib/bot/service/mock.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Bot.Service.Mock do
```

Comment:
How about making `Bot.Service` a behaviour with required callbacks? Of course we don't need it for properly working project.